### ISSUE TITLE
"Invalid Argument: volume"

### ISSUE THREAD

**Issue by tallero** [nothingisdead/npm-kraken-api#36]
Still using version of 28 june, anyway I'm getting "Error: Kraken API returned error: General:Invalid arguments:volume" whenever I try to set an order.
Funny thing is that this happens even if I don't include "volume" field in the json during the API call.

**Comment by aastein** [comment]
try the same order on the the site and check if you get an error.

**Comment by tallero** [comment]
It is kraken's fault. Very unprofessional.
https://www.reddit.com/r/BitcoinMarkets/comments/6pvh4c/kraken_minimum_volume_issue/

**Comment by aastein** [comment]
Yep, ran into same thing... Hope their new backend will solve the issues.
